activities/recruitment/hiring-process.md

```diff
@@ -1,36 +1,37 @@
-8. HRM confirms receipt to candidates and does first ‘sorting out’.
-9. HRM brings possibly suitable candidates to the attention of CEO en supervisor. These 2 make +/- judgement, without consulting each other. If judgement is not equal, a short discussion follows and judgement is made.
-10. HRM notifies candidates (thank you or invitation for interview).
-11. First interview: CEO and HRM, 1 hr, partially on the basis of a pre-established list of desirable competences and other characteristics (i.e. fit for role, fit for organisation (culture, growth, leadership, long/short term)). Half of the interview is spent on presenting PublicCode.net, in the other half, the candidate has a change to present herself/himself. Immediately after the interview both HRM and CEO sore the candidate on the items on the list and determine if the candidate should progress to a second interview.
-12. Second interview: COO (same person as above) with HRM, also partially based on a pre-established list, and with attention to the particularities and matters not discussed in the first interview. Approximately. 80-90% of the time is spent on questioning the candidate. Immediately  after the interview both M and B/C score the candidate, based on the list and other impressions gathered. HRM takes this input to CEO and HRM and CEO determine if the candidate should receive an offer.
-13. CEO & HRM determination of the offer and the bandwidth of the conditions.
-14. Third interview, hopefully a formality, of the preferred candidate with a BOD member (of the legal entity hiring).
-15. HRM makes the offer to the candidate and negotiates, always in consultation with CEO and the director(s).
-16. Signing of contract and starting by the new team member, supervised by their hierarchical supervisor. 
+
+## Announcing job opening
+
+1\. Publish job posting on [Publiccode.net](https://publiccode.net) website
+2\. Bring job opening to the attention of network of current team (via email, LinkedIn, social media)
+3\. Increase awareness of job opening via (specialized) sites and possibly recruiters
+
+## Processing reactions of people expressing interest
+
+4\. Questions about the content of the position: route to CEO for answering
+5\. Questions about the hiring procedure and the conditions: route to HRM for answering
+6\. Applications are sent to HRM, for processing
+
+## Planned recruitment and hiring process
+
+7\. Current job openings have no closing date and applications are considered on a rolling basis. When we find a suitable candidate, we hire and close the opening. This means applications are considered ‘by themselves’ and not ‘in competition’ with a certain set of other applications.
+8\. HRM confirms receipt to candidates and does first round of evaluation.
+9\. HRM brings possibly suitable candidates to the attention of CEO and supervisor. These 2 make +/- judgement, without consulting each other. If judgement is not equal, a short discussion follows and judgement is made.
```

Comment:
@MirjamvT what is meant by 'supervisor' here?

Reply:
the person who is hierarchically above the person ('leidinggevende')